### PR TITLE
feat(app): solution for closures

### PR DESCRIPTION
**Pull request by ellaboevans**
This pull request contains the solution for the labs on Closures

### CODE REVIEW

**Comment by fredrickyalley** [comment]
make the PR title start with small letter
change Solution to solution

**Comment by fredrickyalley** [comment]
I was getting undefined when i click the button

File: week_2/closures/main.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  age: 18,
+  greet() {
+    console.log(
+      `Hello, my name is ${this.name} and I'm ${this.age} years old.\n`
+    );
+  },
+};
+
+// direct call
+// Person.greet();
```
Comment (fredrickyalley) [inline week_2/closures/main.js:12]: remove the commented codes if not needed

**Comment by ellaboevans** [comment]
> make the PR title start with small letter change Solution to solution

Sure, doing that right now

File: week_2/closures/main.js (hunk 1 above)
Comment (ellaboevans) [reply week_2/closures/main.js:12]: Yes please.

Comment (ellaboevans) [reply week_2/closures/main.js:12]: This is done.

**Review by fredrickyalley — APPROVED** [review]

**Pull request merged**